packages/logger_crispin_transport/LICENSE

```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```

Comment:
Will the main license auto forward to the child one?

Reply:
Oh shoot I didn't see that! Good catch